composer.json

```diff
@@ -44,46 +44,47 @@
         }
     ],
     "require": {
-        "composer/installers": "^1.2",
-        "cweagans/composer-patches": "^1.6",
-        "drupal-composer/drupal-scaffold": "^2.2",
+        "composer/installers": "^1.7",
+        "cweagans/composer-patches": "^1.7",
         "drupal-composer/drupal-security-advisories": "8.x-dev",
-        "drupal/admin_toolbar": "^2.2",
-        "drupal/config_split": "^1.3",
-        "drupal/core": "9.0.2 as 8.9.1",
+        "drupal/admin_toolbar": "^2.4",
+        "drupal/config_split": "^1.7",
+        "drupal/core-composer-scaffold": "^9",
+        "drupal/core-recommended": "^9",
         "drupal/crop": "^2.1",
-        "drupal/ctools": "^3.0",
+        "drupal/ctools": "^3.4",
         "drupal/devel": "^4.0",
         "drupal/environment_indicator": "^4.0",
-        "drupal/eu_cookie_compliance": "^1.2",
-        "drupal/honeypot": "^1.27",
-        "drupal/metatag": "^1.3",
+        "drupal/eu_cookie_compliance": "^1.13",
+        "drupal/honeypot": "^2.0",
+        "drupal/metatag": "^1.14",
         "drupal/paragraphs": "^1.12",
-        "drupal/pathauto": "^1.0",
-        "drupal/rabbit_hole": "^1.0",
-        "drupal/radix": "4.x-dev",
-        "drupal/redirect": "^1.0@beta",
-        "drupal/reroute_email": "^1.2",
-        "drupal/simple_sitemap": "^2.11",
-        "drupal/stage_file_proxy": "^1.0",
-        "drupal/styleguide": "^1.0@alpha",
+        "drupal/pathauto": "^1.8",
+        "drupal/rabbit_hole": "^1.0@beta",
+        "drupal/radix": "^4.10",
+        "drupal/redirect": "^1.6",
+        "drupal/simple_sitemap": "^3.8",
+        "drupal/stage_file_proxy": "^1.1",
+        "drupal/styleguide": "^2.0@beta",
         "drupal/ultimate_cron": "^2.0@alpha",
-        "drupal/webform": "^6.0",
-        "drush/drush": "^10.0",
+        "drupal/webform": "^6.0@alpha",
+        "drush/drush": "^10.3",
         "edgedesign/phpqa": "^1.23"
     },
     "require-dev": {
         "bex/behat-step-time-logger": "^2.0",
-        "drupal/coder": "~8.3",
-        "jcalderonzumba/gastonjs": "~1.0.2",
-        "jcalderonzumba/mink-phantomjs-driver": "~0.3.1",
+        "consolidation/robo": "^1",
```

Comment:
Please consider this suggestion:
```suggestion
        "consolidation/robo": "^1",
```

This package is not necessary as is only required by dependencies, but not by us. Looks like it has been added to prevent a conflict requiring the packages, but in this case that problem won't happen. The only use case when there are conflicts is when the composer.lock exists , but the boilerplate does not have composer.lock.